lib/helpers.js

```diff
@@ -90,6 +91,28 @@ function getIMEListFromOutput (stdout) {
   return engines;
 }
 
+async function getSdkToolsVersion() {
+    return new Promise((resolve) => {
+        const androidHome = process.env.ANDROID_HOME;
```

Comment:
I'd rather check first whether this env var is defined and show appropriate error if not